apps.toml

```
@@ -2743,6 +2743,14 @@ state = "working"
 subtags = [ "wiki" ]
 url = "https://github.com/YunoHost-Apps/mediawiki_ynh"
 
+[meet]
+added_date = FIXME
+category = "communication"
+level = 6
+potential_alternative_to = [ "BigBlueButton", "Skype", "Zoom" ]
+state = "working"
+subtags = [ "meeting" ]
+
```

```suggestion
[meet]
category = "communication"
potential_alternative_to = [ "BigBlueButton", "Skype", "Zoom" ]
state = "working"
subtags = [ "meeting" ]
```
Level and added_datewill be set automatically, so no need to write them yet :)